drivers/vulkan/vulkan_context.h

```diff
@@ -264,28 +262,28 @@ class VulkanContext {
 
 	virtual bool _use_validation_layers();
 
-	Error _get_preferred_validation_layers(uint32_t *count, const char *const **names);
+	Error _get_preferred_validation_layers(uint32_t *count, const char *const **names) const;
 
 public:
 	// Extension calls
-	VkResult vkCreateRenderPass2KHR(VkDevice device, const VkRenderPassCreateInfo2 *pCreateInfo, const VkAllocationCallbacks *pAllocator, VkRenderPass *pRenderPass);
-
-	uint32_t get_vulkan_major() const { return vulkan_major; };
-	uint32_t get_vulkan_minor() const { return vulkan_minor; };
-	SubgroupCapabilities get_subgroup_capabilities() const { return subgroup_capabilities; };
-	MultiviewCapabilities get_multiview_capabilities() const { return multiview_capabilities; };
-	VRSCapabilities get_vrs_capabilities() const { return vrs_capabilities; };
-	ShaderCapabilities get_shader_capabilities() const { return shader_capabilities; };
-	StorageBufferCapabilities get_storage_buffer_capabilities() const { return storage_buffer_capabilities; };
-
-	VkDevice get_device();
-	VkPhysicalDevice get_physical_device();
-	VkInstance get_instance() { return inst; }
-	int get_swapchain_image_count() const;
+	VkResult vk_create_render_pass2_khr(VkDevice device, const VkRenderPassCreateInfo2 *pCreateInfo, const VkAllocationCallbacks *pAllocator, VkRenderPass *pRenderPass);
```

Comment:
Since this is a wrapper function around real `vkCreateRenderPass2KHR` and it's not a part of Vulkan API, `vk_` and `_khr` are probably redundant.

Reply:
Sounds logical, I will make changes later.